### ISSUE TITLE
Training with an old version of this repository converges much faster.

### ISSUE THREAD

**Issue by petrsmid** [fizyr/keras-retinanet#610]
We have realised the training with an old version of this repository (from 10th Dec 2017 - commit dd687b5d0b97a1ce67876d930bd1669ace1225f4) converges much faster than training with the actual version.

In the current version we experience the regression loss goes down much slower. The training has to run much longer to achieve similar regression loss and recognition compared to the old version.

Here is a comparison using your training script:
Training with the old version - focus on the numbers of regresssion_loss:
https://pastebin.com/YMyPCK77

Training with the same dataset with the actual version:
https://pastebin.com/SFqrxmS1

**Comment by hgaiser** [comment]
True, but it doesn't mean the same ;)

We've had some different implementations of the regression loss. Most noticeably, how do you normalize it. The paper was a bit ambiguous about it but the current implementation is presumably the correct one (it normalizes over the number of foreground anchors).

I'm assuming this will clarify your issue, so I'll close this.
